jclklib/client/connect_msg.hpp

```diff
@@ -20,9 +20,9 @@
 #include <rtpi/condition_variable.hpp>
 #include <rtpi/mutex.hpp>
 
-namespace JClkLibClient
+namespace clkmgr
 {
```

Comment:
replace line 23 & 24 with __CLKMGR_NAMESPACE_BEGIN
and the last } with __CLKMGR_NAMESPACE_END